packit/api.py

```diff
@@ -210,7 +213,22 @@ def update_dist_git(
             )
 
         if commit_title:
-            self.dg.commit(title=commit_title, msg=commit_msg, prefix="")
+            trailers = (
+                [
+                    (
+                        "From-source-git-commit",
```

Comment:
Can we have a constant for these trailer keys since they are used in multiple places?

Reply:
I thought about this, but ultimately I gave up on this, b/c all names for these constants seemed to be super long, and prone to be confusing to the reader.

I did give it a try though in 00d15d3, using just the tokens as the name of the constants, but I'm still not convinced why and how this would be better than just using the string literals in their own.

I'm open to be convinced, though :)

Reply:
For me it's always about:
* avoid possible typos
* reference a constant, not a string, especially when it's long